tests/consent_mailer/test_main.py

```diff
@@ -1,5 +1,115 @@
+import base64
+from typing import Any
+
+import pytest
+import requests
+
+from mex.common.models import MergedPerson
+from mex.common.types import AssetsPath
+from mex.extractors.consent_mailer.main import (
+    send_consent_emails_to_persons,
+)
 from mex.extractors.pipeline import run_job_in_process
+from mex.extractors.settings import Settings
 
 
+@pytest.mark.usefixtures("mocked_consent_backend_api_connector")
+@pytest.mark.integration
 def test_job() -> None:
     assert run_job_in_process("consent_mailer")
+
+
+def _req_header() -> dict[str, str]:
+    settings = Settings.get()
+    return {
+        "Authorization": "Basic "
+        + base64.b64encode(
+            f"{settings.consent_mailer.mailpit_api_user.get_secret_value()}:{settings.consent_mailer.mailpit_api_password.get_secret_value()}".encode()
+        ).decode()
+    }
+
+
+def _req_verify() -> bool | str:
+    settings = Settings.get()
+    if isinstance(settings.verify_session, AssetsPath):
+        return settings.verify_session._path.absolute().as_posix()
+    return settings.verify_session
+
+
+def _delete_messages() -> None:
+    settings = Settings.get()
+    requests.delete(
+        f"{settings.consent_mailer.mailpit_api_url}/api/v1/messages",
+        timeout=3,
+        verify=_req_verify(),
+        headers=_req_header(),
+    )
+
+
+def _get_messages() -> Any:  # noqa: ANN401
+    settings = Settings.get()
+    return requests.get(
+        f"{settings.consent_mailer.mailpit_api_url}/api/v1/messages",
+        timeout=3,
+        verify=_req_verify(),
+        headers=_req_header() | {"accept": "application/json"},
```

Comment:
```suggestion
        auth=(
            settings.consent_mailer.mailpit_api_user.get_secret_value(),
            settings.consent_mailer.mailpit_api_password.get_secret_value()
        ),
```

accept header is not really needed, and requests can do basic-auth out of the box with a (user, pw) tuple